src/cdi_analytic/test/tstCompound_Analytic_MultigroupOpacity.cc

```diff
@@ -135,11 +135,11 @@ void multigroup_test(rtt_dsxx::UnitTest &ut) {
 
   // scalar density/temperature + vector density/temperature
   vector<double> data_field(3, 2.0);
-  vector<vector<double>> sig_t = opacity.getOpacity(data_field, 3.0);
+  vector<vector<double>> sig_test = opacity.getOpacity(data_field, 3.0);
```

Comment:
@timmah Can you change this variable name to `sigma_t` or `sigma_total`?  Thanks!